Symfony/CS/Tests/Fixer/Symfony/NewWithBracesFixerTest.php

```diff
@@ -38,15 +38,42 @@ public function test54($expected, $input = null)
     public function provideStandardCases()
     {
         return array(
-            array('<?php $x = new X();', '<?php $x = new X;'),
-            array('<?php $y = new Y() ;', '<?php $y = new Y ;'),
-            array('<?php $foo = new $foo();', '<?php $foo = new $foo;'),
-            array('<?php $baz = new {$bar->baz}();', '<?php $baz = new {$bar->baz};'),
-            array('<?php $xyz = new X(new Y(new Z()));', '<?php $xyz = new X(new Y(new Z));'),
-            array('<?php $foo = (new $bar())->foo;', '<?php $foo = (new $bar)->foo;'),
-            array('<?php $self = new self();', '<?php $self = new self;'),
-            array('<?php $static = new static();', '<?php $static = new static;'),
-            array('<?php $magic = new __CLASS__();', '<?php $magic = new __CLASS__;'),
+            array(
+                '<?php $x = new X();',
+                '<?php $x = new X;',
+            ),
+            array(
+                '<?php $y = new Y() ;',
+                '<?php $y = new Y ;',
+            ),
+            array(
+                '<?php $foo = new $foo();',
+                '<?php $foo = new $foo;',
+            ),
+            array(
+                '<?php $baz = new {$bar->baz}();',
+                '<?php $baz = new {$bar->baz};',
+            ),
+            array(
+                '<?php $xyz = new X(new Y(new Z()));',
+                '<?php $xyz = new X(new Y(new Z));',
+            ),
+            array(
+                '<?php $foo = (new $bar())->foo;',
+                '<?php $foo = (new $bar)->foo;',
+            ),
+            array(
+                '<?php $self = new self();',
+                '<?php $self = new self;',
+            ),
+            array(
+                '<?php $static = new static();',
+                '<?php $static = new static;',
+            ),
+            //array( was this valid in any PHP version? https://bugs.php.net/bug.php?id=36221
```

Comment:
Nope, this is OK:
```php
<?php

class Foo
{
    public static function create()
    {
        $x = __CLASS__;
        return new $x();
    }
}

echo get_class(Foo::create());
```

but this is not:
<?php

class Foo
{
    public static function create()
    {
        return new __CLASS__();
    }
}

echo get_class(Foo::create());